fe/fe-core/src/main/java/com/starrocks/sql/plan/PlanFragmentBuilder.java

```diff
@@ -1663,20 +1663,6 @@ private PlanFragment buildSetOperation(OptExpression optExpr, ExecPlan context,
                 setOperationNode = new UnionNode(context.getPlanCtx().getNextNodeId(), setOperationTuple.getId());
                 isUnionAll = ((PhysicalUnionOperator) setOperation).isUnionAll();
                 setOperationNode.setFirstMaterializedChildIdx_(optExpr.arity());
-
-                List<Map<Integer, Integer>> passThroughSlotMaps = new ArrayList<>();
```

Comment:
Why change this?

Reply:
Because to push down RuntimeFilter to children nodes, we have to know the mapping from output-column-ref to column-ref-in-child.

And right now this mapping is only constructed in UnionNode, and if we want to push down RF to ExceptNode/IntersectNode, we have to construct this mapping in those nodes too.

Fortunately I __think__ this logic is the same for UnionNode/IntersectNode/ExceptNode, so I put this piece of code outside.